examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py

```diff
@@ -77,58 +90,41 @@
+                        'and your dataset builder args: args.pad_max_length.')
+
 # =============AWQ configs====================
 parser.add_argument("--use_auto_scale", action="store_true",
                     help="Enables best scales search based on activation distribution.")
 parser.add_argument("--use_auto_clip", action="store_true",
-                    help="Enables clip range searchc.")
+                    help="Enables clip range search.")
 parser.add_argument("--folding", action="store_true",
                     help="Allow insert mul before linear when the scale cannot be absorbed by last layer for TEQ/AWQ.")
 parser.add_argument('--absorb_layer_dict', type=dict, default={},
                     help="The layer dict that scale can be absorbed for TEQ/AWQ.")
+
 # ============AUTOROUND configs==============
-parser.add_argument(
-    "--lr",
-    type=float,
-    default=None,
-    help="learning rate, if None, it will be set to 1.0/iters automatically",
-)
-parser.add_argument(
-    "--minmax_lr",
-    type=float,
-    default=None,
-    help="minmax learning rate, if None,it will beset to be the same with lr",
-)
-parser.add_argument("--autoround_iters", default=200, type=int, help="num iters for autoround calibration.")
-parser.add_argument("--autoround_nsamples", default=128, type=int, help="num samples for autoround calibration.")
-parser.add_argument(
-    "--disable_quanted_input",
-    action="store_true",
-    help="whether to use the output of quantized block to tune the next block",
-)
+parser.add_argument("--lr", type=float, default=None,
+                    help="Learning rate, if None, it will be set to 1.0/iters automatically")
+parser.add_argument("--minmax_lr", type=float, default=None,
+                    help="Minmax learning rate, if None,it will beset to be the same with lr")
+parser.add_argument("--Autoround_iters", default=200, type=int,
+                    help="Num iters for autoround calibration.")
+parser.add_argument("--autoround_nsamples", default=128, type=int, 
+                    help="Num samples for autoround calibration.")
+parser.add_argument("--disable_quanted_input", action="store_true",
+                    help="Whether to use the output of quantized block to tune the next block",)
 
 # =============DoubleQuant configs====================
-parser.add_argument("--double_quant_type",
-                    type=str,
-                    default=None,
+parser.add_argument("--double_quant_type", type=str, default=None,
                     choices=['GGML_TYPE_Q4_K', 'BNB_NF4'],
-                    help="DoubleQuant parameter")
-parser.add_argument("--double_quant_dtype",
-                    type=str,
-                    default="fp32",
+                    help="DoubleQuant parameter.")
+parser.add_argument("--double_quant_dtype", type=str, default="fp32",
                     help="Data type for double quant scale.")
```

Comment:
We should notice here that double quant is a fake quant for accuracy emulation.

How about only keeping opt-125m double_quant validation and remove other models. It's a usage that is not popular, and HPU device doesn't support it currently. @XuehaoSun 
BTW, we need to update run_quant.sh to correct `int4_dp_bnb` -> `nf4_dp_bnb`

Reply:
https://github.com/intel/neural-compressor/pull/2053/commits/a61e9854a3501f8a2f0e9ee0241d724540b8edc5
@XuehaoSun please check.

Reply:
since double quant is not applied to HPU, we should have some way to let user know it so as to avoid user use it on Gaudi

Reply:
Let's notice user in the readme @Kaihui-intel

Reply:
added in notes https://github.com/intel/neural-compressor/pull/2053/commits/9140d24689e5adffb10953cfef16d78f7a9acaa3